XenAdmin/Wizards/NewVMWizard/Page_CloudConfigParameters.cs

```diff
@@ -129,11 +152,85 @@ private void GetDefaultParameters()
             var result = action.Result.Replace("\n", Environment.NewLine);
 
             ConfigDriveTemplateTextBox.Text = result;
+            existingConfig = result;
+        }
+
+        private void GetDefaultParameters()
+        {
+            GetCloudConfigParameters(null);
+        }
+
+        public void ShowHideButtonsAndWarnings(bool inNewVmWizard)
+        {
+            // IncludeConfigDriveCheckBox and reloadDefaults only visible in the New VM Wizard
+            IncludeConfigDriveCheckBox.Visible = reloadDefaults.Visible = inNewVmWizard;
```

Comment:
I would suggest IncludeConfigDriveCheckBox to be visible, ticked, but greyed out